genesis_core/user_api/iam/dm/models.py

```diff
@@ -532,6 +532,23 @@ def get_default(cls, organization=None, user=None):
                 return role_binding.project
         return None
 
+    @classmethod
+    def list_my(cls, filters=None):
+        user = User.me()
+        filters = filters or {}
+        filters.update({"user": ra_filters.EQ(user)})
+        role_bindings = RoleBinding.objects.get_all(
+            filters=filters,
+            order_by={"created_at": "asc"},
+        )
+        return [
+            binding.project for binding in role_bindings if binding.project
```

Comment:
Minor - we may filter empty projects in DB by `"project": ra_filters.IsNot(None)`

And - do we need to sort them by default? created_at doesn't have index. But I don't see it as a problem, it's ok to leave sorting

Reply:
done